csaf_files/IT/white/2024/va-24-201-01.json

```diff
@@ -319,8 +320,9 @@
       "acknowledgments": [
         {
           "names": [
-            "Matthew Galligan, CISA Rapid Action Force"
-          ]
+            "Matthew Galligan"
+          ],
+          "organization": "CISA Rapid Action Force",
         }
```

Comment:
See above comment:
https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#311-acknowledgments-type

Reply:
I agree with the above change.